### PR TITLE
feat: add close market instruction

### PR DESCRIPTION
**Pull request by zaxozhu**

### CODE REVIEW

**Review by olgkv — APPROVED** [review]

File: membership-token/js/test/actions/close-market.ts [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+  connection,
+  market,
+}: CloseMarketParams): Promise<void> => {
+  const instruction = await createCloseMarketInstruction({
+    market: market.publicKey,
+    owner: payer.publicKey,
+    clock: SYSVAR_CLOCK_PUBKEY,
+  });
+
+  const marketTx: Transaction = await createAndSignTransaction(
```
Comment (zaxozhu) [inline membership-token/js/test/actions/close-market.ts:36]: Let's make this method to return transaction and remove sending of it from here. That way we can remove closeMarketLimitedDuration

File: membership-token/js/test/close-market.test.ts [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+  await sleep(3000);
+
+  console.log('Payer: ', payer.publicKey.toBase58());
+  await closeMarket({
+    test: t,
+    transactionHandler,
+    payer,
+    connection,
+    market,
+  });
```
Comment (zaxozhu) [inline membership-token/js/test/close-market.test.ts:76]: Let's call send here

```
const MarketRes = await transactionHandler.sendAndConfirmTransaction(
    marketTx,
    [payer],
    defaultSendOptions,
  );
```

File: membership-token/js/test/close-market.test.ts [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+    connection,
+    store: store.publicKey,
+    sellingResource: sellingResource.publicKey,
+    treasuryMint: treasuryMint.publicKey,
+    params,
+  });
+
+  await sleep(3000);
+
+  await closeMarketLimitedDuration({
```
Comment (zaxozhu) [inline membership-token/js/test/close-market.test.ts:132]: const closeMarketTx = await closeMarket({

```
const MarketRes = await transactionHandler.sendAndConfirmTransaction(
    marketTx,
    [payer],
    defaultSendOptions,
  );
```

File: membership-token/js/test/actions/index.ts [hunk 4]
```diff
@@ -3,3 +3,7 @@ export * from './create-prerequisites';
 export * from './create-store';
 export * from './mint-nft';
 export * from './init-selling-resource';
+export * from './create-market';
+export * from './close-market';
+
+
```
Comment (zaxozhu) [inline membership-token/js/test/actions/index.ts:9]: Remove these spaces pls

**Comment by zaxozhu** [comment]
@olgkv added some comments

File: membership-token/js/test/actions/close-market.ts [hunk 5]
```diff
@@ -0,0 +1,32 @@
+import { Connection, Keypair, SYSVAR_CLOCK_PUBKEY, Transaction } from '@solana/web3.js';
```
Comment (zaxozhu) [inline membership-token/js/test/actions/close-market.ts:1]: Move it to /transactions please

**Pull request closed without merging**